### DISCUSSION TITLE
How do we customize the draghandle template?

### DISCUSSION THREAD

**Discussion by yellow1912** [ueberdosis/tiptap#5712] [General Questions]
I really like this example here:

https://templates.tiptap.dev/NjHMLKTtdW

I wonder what is the proper way to customize this extension exactly like the example above? Should we use the render method? If so how should we hook the trigger to toggle custom context menu?

I'm so excited that we get this new feature after such a long wait.

**Comment by kart-c** [comment]
If you are using React, you can refer the [answer here](https://github.com/ueberdosis/tiptap/discussions/5637#discussioncomment-10738515).

If you are using Vue, I haven't used it extensively, but I can try and figure it out if you can provide a sandbox for the same.
